File: Data Structures & Algorithms/missing-number/submission-9.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int porownanie = 0;
        sort(nums.begin(), nums.end());
        for(int i = 0; i<nums.size(); i++){
            if(porownanie != nums[i]){
                return porownanie;
            }
            porownanie++;
        }
    }
};

File: Data Structures & Algorithms/top-k-elements-in-list/submission-9.cs
public class Solution {
    public int[] TopKFrequent(int[] nums, int k) {
        var map = new Dictionary<int, int>();


        foreach(int number in nums){
            if(!map.ContainsKey(number)){
                map[number] = 0;
            }
            map[number]++;
        }
        return map.OrderByDescending(x => x.Value) .Take(k) .Select(x => x.Key) .ToArray();
    }
}
